dNet/dServer.cpp

```diff
@@ -191,6 +193,11 @@ bool dServer::Startup() {
 	return true;
 }
 
+void dServer::UpdateBandwidthLimit() {
+	auto newBandwidth = mConfig->GetValue("maximum_outgoing_bandwidth");
+	mPeer->SetPerConnectionOutgoingBandwidthLimit(!newBandwidth.empty() ? std::stoi(newBandwidth) : 0);
```

Comment:
Does 0 mean unlimited?

Reply:
yes - 0 is unlimited by default as per RakNet documentation

Reply:
cool beans